pyfarm/master/entrypoints.py

```diff
@@ -1,6 +1,7 @@
 # No shebang line, this module is meant to be imported
 #
 # Copyright 2013 Oliver Palmer
+# Copyright 2014, 2015 Ambient Entertainment GmbH & Co. KG
```

Comment:
I've seen it done many different ways over the years, including the way you're suggesting here, only applying it for the first year (which is what I've been doing), or a single copyright notice in a central file.

Anyway, I'm going to merge though here's some of the reasons I've been told over the years why this change would not be strictly necessary by people who are more familiar with the laws:

* When a work is created or edited it's protected under copyright law even without the declaration.  Based on some of my own research, it used to be that a line stating the copyright was required but the Berne Convention Implementation Act of 1989 seems to have removed this as a requirement.  It also seems that as a result getting the copyright incorrect also does not invalidate the copyright.
* Copyright formalities, such as the line declaring the year, are more about when the copyright is supposed to expire.  It depends on how the work was created and by who of course but it's been [extended](http://en.wikipedia.org/wiki/Copyright_Term_Extension_Act#mediaviewer/File:Copyright_term.svg) several times over.
* The copyright line is encouraged in various parts of the laws, mainly so claims of infringement can be dealt with.
* [17 U.S.C. § 401](http://www.copyright.gov/title17/92chap4.html), which is one of the places the copyright header is encouraged, states to use the first year of publication.  It also states the year can be omitted.

I'm not a lawyer of course, but based on what we've seen out there and some of the information above it seems that both points of view are supported.  But in the interest of consistency and because it helps to easily determine the 'age' of a file I support this change.  I also think we should probably go ahead and update the existing code so we're internally consistent (I might do that this weekend).